src/stash-list/index.vue

```diff
@@ -163,6 +164,25 @@ export default defineComponent({
             return `Search ${counts.groups} ${groups}, ${counts.tabs} ${tabs}`;
         },
 
+        tabStats(): { open: number, discarded: number, hidden: number } {
+            let open = 0, discarded = 0, hidden = 0;
+            for (const tab of this.tabs) {
+                if (tab.hidden) {
+                    hidden += 1;
+                } else if (tab.discarded) {
+                    discarded += 1;
+                } else {
+                    open += 1;
+                }
+            }
+            return { open, discarded, hidden };
+        },
+
+        tooltip(): string {
+            const st = this.tabStats;
+            return `(groups: ${this.counts.groups}, stashed: ${this.counts.tabs}, tabs: ${st.open}/${st.discarded}/${st.hidden} open/unloaded/hidden)`;
```

Comment:
Same as above, I think a more natural-language syntax would be better, and since this is the entire content of the tooltip, I'd probably rearrange the parentheses a bit.

Also, it might be good to draw a distinction between the total number of open tabs and the number of visible tabs, maybe something like:

```
N groups, M stashed tabs
W open tabs (X loaded, Y unloaded, Z hidden)
```

where W = X+Y+Z.

Reply:
Sounds reasonable. Done.